channeldb/invoices.go

```diff
@@ -1187,6 +1199,7 @@ func deserializeHtlcs(r io.Reader) (map[CircuitKey]*InvoiceHTLC, error) {
 		htlc.ResolveTime = time.Unix(0, int64(resolveTime))
 		htlc.State = HtlcState(state)
 		htlc.Amt = lnwire.MilliSatoshi(amt)
+		htlc.MppTotalAmt = lnwire.MilliSatoshi(mppTotalAmt)
```

Comment:
It should be ok. If it is a non-mpp htlc, the way to signal that is with a zero total amount. Only in case we would have accepted an mpp htlc which isn't resolved yet, we may discard the mpp fields here. But previously we only held htlcs for hodl invoices and once an invoice is in the accepted state, the mpp logic is no longer exercised.

We probably should have been stricter when we started parsing mpp fields in a previous pr (without the implementation being complete) by rejecting those htlcs.